saml-based-authentication/README.md

```diff
@@ -0,0 +1,35 @@
+# SAML Integration with Media Shuttle SDK
+
+This documentation specifies steps to integrate SAML with Media Shuttle SDK using [Okta Workforce Identity Cloud](https://www.okta.com) and [AWS Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html).
+
+## Create a SAML app in Okta
+
+- Create an [Okta Workforce Identity](https://www.okta.com/free-trial/workforce-identity/) account and sign in.
+- Create [SAML app integration](https://help.okta.com/en-us/content/topics/apps/apps_app_integration_wizard_saml.htm).
+- Under `GENERAL`, for Single sign on URL, enter `https://yourDomainPrefix.auth.region.amazoncognito.com/saml2/idpresponse`. Replace `yourDomainPrefix` and `region` with the values for your `user pool`.
```

Comment:
Is there any prerequisite? where does admin need to have in place before getting these values?

Reply:
Refactored ReadMe with diagram and in a better-structured way, added prerequisite as well